### DISCUSSION TITLE
Account currency

### DISCUSSION THREAD

**Discussion by tomekwl** [ghostfolio/ghostfolio#4040] [Q&A] [answered]
I run Ghostfolio under Umbrel. The only account currencies I see are USD and USX. Hosted version has them all. What am I missing?

**Comment by dtslvr** [comment, marked as answer]
Hi @tomekwl

Please check the self-hosting [FAQ section](https://ghostfol.io/en/faq/self-hosting): _How do I add a new currency?_

After a refresh, you can select the currency in your account.

**Reply by Dgram19** [reply]
I had already read this thread, only I don't even in admin control it allows me to add currencies, that's why I had created an additional topic. Thank you
